pom.xml

```
@@ -6,7 +6,7 @@
 
   <groupId>com.nablarch.integration</groupId>
   <artifactId>nablarch-micrometer-adaptor</artifactId>
-  <version>5-NEXT-SNAPSHOT</version>
+  <version>1.2.0</version>
```

5u25(5-NEXT-SNAPSHOT)であるべきですが
外部貢献者は5-NEXT-SNAPSHOTにアクセスできません。
ビルド時に支障があるため最後に取り込みます
```suggestion
  <version>5-NEXT-SNAPSHOT</version>
```

---

@arukiidou 
仰る通りここは`5-NEXT-SNAPSHOT`のままとしてほしいです。
外部環境でのSNAPSHOTのビルドですが、Mavenのsettings.xmlで`repository ` `pluginRepository`にURL `https://oss.sonatype.org/content/repositories/snapshots` を追加することで可能になります。
`repositories`の設定例を以下に記載しますので、お試しいただけますでしょうか。

```xml
<repositories>
    <repository>
        <id>sonatype-snapshot</id>
        <name>Sonatype Snapshot</name>
        <url>https://oss.sonatype.org/content/repositories/snapshots</url>
        <snapshots>
            <enabled>true</enabled>
        </snapshots>
    </repository>
</repositories>
```

---

承知しました。他の指摘を全て解決したら取り込みます。

---

対応しました。